libs/postgres_ffi/src/pg_constants.rs

```diff
@@ -55,6 +56,23 @@ pub const TRANSACTION_STATUS_SUB_COMMITTED: u8 = 0x03;
 pub const CLOG_ZEROPAGE: u8 = 0x00;
 pub const CLOG_TRUNCATE: u8 = 0x10;
 
+//
+// Constants from csn_log.c, csn_log.h, and csn_snapshpot.h
+// 
+
+pub const CSN_LOG_XACTS_PER_PAGE: u32 = BLCKSZ as u32 / std::mem::size_of::<XidCSN>() as u32;
```

Comment:
Could you make the `std::mem::size_of::<XidCSN>()` a constant in this file then replace all the use of that with the new constant?

Reply:
Done.